tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/arrays/SealedArrayTest.java

```diff
@@ -36,177 +36,117 @@
     }
 
     @Test
     public void testCreateBooleanSealedArray() {
-        BValue[] returnValues = BRunUtil.invoke(compileResult, "createBoolSealedArray");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(((BInteger) returnValues[0]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createBoolSealedArray");
 
-        returnValues = BRunUtil.invoke(compileResult, "createBoolSealedArrayWithLabel");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(((BInteger) returnValues[0]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createBoolAutoFilledSealedArray");
 
-        returnValues = BRunUtil.invoke(compileResult, "createBoolDefaultSealedArray");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(returnValues[0].stringValue(),
-                "[false, false, false, false, false]", "Length didn't match");
-        Assert.assertEquals(((BInteger) returnValues[1]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createBoolSealedArrayWithLabel");
+
+        BRunUtil.invoke(compileResult, "createBoolDefaultSealedArray");
     }
 
     @Test
     public void testCreateFloatSealedArray() {
-        BValue[] returnValues = BRunUtil.invoke(compileResult, "createFloatSealedArray");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(((BInteger) returnValues[0]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createFloatSealedArray");
 
-        returnValues = BRunUtil.invoke(compileResult, "createFloatSealedArrayWithLabel");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(((BInteger) returnValues[0]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createFloatAutoFilledSealedArray");
 
-        returnValues = BRunUtil.invoke(compileResult, "createFloatDefaultSealedArray");
-        Assert.assertFalse(
-                returnValues == null || returnValues.length == 0 || returnValues[0] == null, "Invalid Return Values.");
-        Assert.assertEquals(returnValues[0].stringValue(), "[0.0, 0.0, 0.0, 0.0, 0.0]", "Length didn't match");
-        Assert.assertEquals(((BInteger) returnValues[1]).intValue(), 5, "Length didn't match");
+        BRunUtil.invoke(compileResult, "createFloatSealedArrayWithLabel");
+
```

Comment:
Shall we remove the extra lines?

Reply:
Fixed https://github.com/ballerina-platform/ballerina-lang/pull/20749/commits/ab2afb7227959b618fcd4d99c0854f2ca5b1dd40